mkosi/distributions/gentoo.py

```diff
@@ -211,43 +112,58 @@ def fetch_fix_stage3(self) -> None:
             else:
                 die("profile names changed upstream?")
 
+        stage3_tar_path = state.cache_dir / stage3_tar
+        cls.stage3_cache = stage3_tar_path.with_name(stage3_tar.name).with_suffix(".tmp")
+
+        user_config_path = cls.stage3_cache / USER_CONFIG_PATH
+
+        emerge_vars = {
+            "FEATURES": " ".join([
+                    # -user* are required for access to USER_CONFIG_PATH
+                    "-userfetch",
+                    "-userpriv",
+                    "-usersync",
+                    "-usersandbox",
+                    "-sandbox",
+                    "-pid-sandbox",  # for cross-compile scenarios
+                    "-network-sandbox",
```

Comment:
Do we need all these if `bwrap` is already providing the sandbox? Seems like doing double work for no benefit

Reply:
`-sandbox' reads dont-do-sandbox, hyphene '-' prefix is for _minus_.